lib/models/room.js

```diff
@@ -527,3 +557,14 @@ module.exports = Room;
  *   var newName = room.name;
  * });
  */
+
+/**
+ * Fires whenever a room's tags are updated.
+ * @event module:client~MatrixClient#"Room.tags"
+ * @param {Room} room The room whose Room.tags was updated.
+ * @example
+ * matrixClient.on("Room.tags", function(room){
```

Comment:
Given this does have a 1:1 mapping of "we got some event and then fired this thing", it would be polite to include the `MatrixEvent` which made this event fire as the first arg for consistency with other events which can fire (mostly `RoomState` events).